File: task 11/testXMLDICT/testXMLDICT/ViewController.m
//
//  ViewController.m
//  testXMLDICT
//
//  Created by 张皓 on 16/8/5.
//  Copyright © 2016年 张皓. All rights reserved.
//

#import "ViewController.h"
#import "XMLDictionary.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
    NSXMLParser *paser = [[NSXMLParser alloc] initWithContentsOfURL:[NSURL fileURLWithPath:[[NSBundle mainBundle] pathForResource:@"questions" ofType:@"xml" ]]];

    NSDictionary *dict = [NSDictionary dictionaryWithXMLParser:paser];
    
    NSLog(@"%@", dict);
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: task 11/AnalyzeXML/AnalyzeXML/ViewController.swift
//
//  ViewController.swift
//  AnalyzeXML
//
//  Created by 张皓 on 16/8/5.
//  Copyright © 2016年 张皓. All rights reserved.
//

import UIKit


class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()

        let paser: NSXMLParser = NSXMLParser.init(contentsOfURL: NSURL.fileURLWithPath(NSBundle.mainBundle().pathForResource("questions", ofType: "xml")!))!
        let dic: NSDictionary = NSDictionary.init(XMLParser: paser)
        
        print(dic)
        
//        
        
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        
    }


}

